src/main/java/com/connexta/store/service/impl/StoreServiceImpl.java

```diff
@@ -216,4 +216,39 @@ public void quarantine(
       adaptor.delete(datasetId);
     }
   }
+
+  @Override
+  public void unstage(
+      @Pattern(regexp = "^[0-9a-zA-Z]+$") @Size(min = 36, max = 36) final String datasetId)
```

Comment:
✏️ This pattern regex is incorrect for a 36 length UUID. These annotations don't even do anything here. I'd rather just not have it.